effectiveOfficeBackend/Dockerfile

```diff
@@ -1,4 +1,15 @@
-FROM gradle:7-jdk11 AS build
+FROM gradle:7-jdk11 AS graalvm-install
+WORKDIR /opt
+ADD https://download.oracle.com/graalvm/17/latest/graalvm-jdk-17_linux-x64_bin.tar.gz graalvm-jdk-17_linux-x64_bin.tar.gz
+RUN tar -xvf graalvm-jdk-17_linux-x64_bin.tar.gz && \
+    mv graalvm-ce-java17-17.3.0 graalvm && \
+    rm graalvm-jdk-17_linux-x64_bin.tar.gz
+
+ENV JAVA_HOME=/opt/graalvm
+ENV PATH=${JAVA_HOME}/bin:${PATH}
+
+#FROM gradle:7-jdk11 AS build
+FROM graalvm-jdk-17_linux-x64_bin.tar.gz AS build
```

Comment:
Docker engine cannot fetch layers from gunzipped tar, only from existing local or remote image

Reply:
It’s better not to reinvent the wheel and download graalvm; to test the concept, it’s enough to use the collected images from [graalvm itself](https://github.com/graalvm/container/pkgs/container/jdk-community)

i. e.
```dockerfile
FROM ghcr.io/graalvm/jdk-community:22.0.1
```

The same thing in the final step, instead of
```dockerfile
FROM openjdk:11
```

Use the graalvm image, otherwise it makes no sense that you end up running the jar on OpenJDK